pkg/webhook/workspace/config.go

```diff
@@ -36,6 +36,7 @@ import (
 )
 
 const certGenDeploymentName = "che-workspace-controller-cert-gen"
+const dummyWebhookConfigurationName = "controller.devfile.io-blocker"
```

Comment:
It's better to avoid duplication. Can we just use the value from cert-generation/config?